Makefile

```diff
@@ -33,10 +34,16 @@ mock: tools
 	mockgen -source pkg/aws/eks.go -destination mock/eks.go -package mock
 	mockgen -source pkg/k8s/jobs.go -destination mock/k8s_jobs.go -package mock
 	mockgen -source pkg/k8s/util.go -destination mock/k8s_util.go -package mock
+	go generate ./...
 
 .PHONY: mock
 
 .PHONY: k8s-integration-test
 k8s-integration-test:
 	echo K8S_VERSION: $(K8S_VERSION)
-	go test -v -timeout 10m -count 1 ./integration/k8s_test.go
\ No newline at end of file
+	go test -v -timeout 10m -count 1 ./integration/k8s_test.go
+
+.PHONY: vault-integration-test
+vault-integration-test:
+	echo VAULT_VERSION: $(VAULT_VERSION)
+	go test -v -timeout 10m -count 1 ./integration/vault_test.go
```

Comment:
We may want to consider build tags in the future to target the execution of specific tests: https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool

In my experience tags like the following allow you to `go test -tags=integration $(SOURCE)`:

```golang
//go:build integration
// +build integration

package main
...
````

Reply:
Interesting... let me look into that some more. There's some cleanup / refactoring work that needs to be done in the library and this seems like a good candidate to include in that.